publication/ver11/cr/security_and_privacy.md

```diff
@@ -0,0 +1,785 @@
+[WoT Security Testing Plan](https://github.com/w3c/wot-security-testing-plan).
+
+In the documents above we address the threats and attack surfaces of the
+WoT in detail.  In the following we have answered the questions posed by the
+W3C Security and Privacy Questionnaire as best as we could... but please take into consideration
+that most of these questions are not fully applicable to our situation.
+
+Security in the IoT is unavoidably more complex than in the web.
+IoT systems often use patterns of communication that are more general
+than the relatively straightforward client-server architecture of web servers and browsers.
+For example, the "same origin policy" is considered a cornerstone of web security.
+However, a single IoT device may connect and integrate information from several other
+IoT devices (and even send commands to them) as a fundamental part of its operation.
+In addition,
+security on the web is often concerned with the "user's device" or the "user agent",
+which is always a single client (the browser).
+The IoT is often more concerned with machine-to-machine communication;
+there is frequently not even a human "user" directly associated with a device.
+If an IoT device _is_ associated with a user, such as in a Smart Home context,
+the device may be a client, or
+a server---or even both at the same time.
+Devices may also be associated with many users, not just one.
+Finally, even if an IoT device has
+a "user interface" it will be more varied than that of the browser,
+and this UI will be under control of the device manufacturer, not the proposed WoT standard.
+
+The kinds of risks we have to deal with in the WoT are also different than
+those in the Web.  For example, unlike HTML the WoT does not directly support downloading
+and executing scripts from other network services.  So many risks having
+to deal with such execution are not relevant.  On the other hand, among other risks, the types
+of sensors and actuators supported by IoT devices are much broader and
+can even include systems whose compromise would have safety implications.
+
+## Questions and Answers
+
+### 1 [What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?](https://www.w3.org/TR/security-privacy-questionnaire/#purpose)
+The following addresses this question for each deliverable:
+* Thing Description (TD): A TD includes information about a Thing's network API
+  for the purpose of interacting with it.
+  This includes a description of security mechanisms used for the purpose of developers 
+  needing to know what is necessary to interact with a Thing.
+  This metadata is non-secret information only, but if TDs are freely available this
+  could allow attackers to scan for vulnerabilities; this is discussed in the "Vulnerability Auditing"
+  Security Consideration in the TD document.
+  One issue is that TDs are descriptive and so can be used to describe insecure interfaces
```

Comment:
Or broken auth.  There is nothing forcing people to use good practices when defining an IoT interface, and in fact I know of lots of examples where less-than-perfect security is used.  Home Assistant, for example, uses "password" over HTTP which is semi-useless, since the password gets transmitted in the clear (within my LAN, protected only by WEP).  So why bother?  However, I can *describe* such an interface using TDs.

Reply:
Propose adding a definition of an "insecure interface":
An insecure interface is one with a known vulnerability.